### ISSUE TITLE
Refactor blender_verification_order task

### ISSUE THREAD

**Issue by dybi** [golemfactory/concent#549]
There are many "code blobs" (ie. downloading archives, unpacking them, runnnig blender, comparing images, etc) in `blender_verification_order` which have just one reponsibility and should be extracted to small, well-named function.
Also in each of this "code blobs" in case of exception, `VerificationResult(ERROR)` is being scheduled. This is a significant code duplication and should be handled by introduction of decorator, which would schedule `VerificationResult(ERROR)` in case any of new samll functions raises `VerificatioinError` (which should be introduced.
**TL;DR**
1. Extract small well named functions from `blender_verification_order`
2. Introduce new exception `VerificationError` which will be rased in new small functions in case of erroneous situation
3. Introduce new decorator `handle_verification_errors` which would catch all `VerificationError` and schedule `verification_result` witg result `ERROR`

**Comment by dybi** [comment]
Summary:
- mypy annotation is missing, separate task has been created: https://github.com/golemfactory/concent/issues/583
- unit tests for helper functions are missing, separate task has been created: https://github.com/golemfactory/concent/issues/582
